src/particles/portal/component.tsx

```diff
@@ -25,31 +25,74 @@ const StyledPortal = styled.div<IStyledPortalProps>`
   ${(props: IStyledPortalProps): string => themeToCss(props.theme.background)};
 `;
 
+export enum Placement {
+  bottomLeft = 'bottom-left',
+  bottomCenter = 'bottom-center',
+  bottomRight = 'bottom-right',
+  topLeft = 'top-left',
+  topCenter = 'top-center',
+  topRight = 'top-right'
+}
+
+export function getOffsetTop(rect: any, vertical: number | string): number {
```

Comment:
Please change the type of the `rect` argument.

Reply:
I was not able to get the type of this arg. The value of this arg is going to be of `anchorElementNodeRect = props.anchorElement?.current?.getBoundingClientRect();`